samples/public/BlankUwpNet9App/BlankUwpNet9App.csproj

```diff
@@ -0,0 +1,41 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>net9.0-windows10.0.26100.0</TargetFramework>
+    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
+    <WindowsSdkPackageVersion>10.0.26100.57</WindowsSdkPackageVersion>
+    <Platforms>x86;x64;arm64</Platforms>
+    <RuntimeIdentifiers>win-x86;win-x64;win-arm64</RuntimeIdentifiers>
+    <PublishProfile>win-$(Platform).pubxml</PublishProfile>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <PublishAot>true</PublishAot>
+    <UseUwp>true</UseUwp>
+    <DisableRuntimeMarshalling>true</DisableRuntimeMarshalling>
+    <EnableMsixTooling>true</EnableMsixTooling>
+	  <UnitTestPlatformVersion Condition="'$(UnitTestPlatformVersion)' == ''">$(VisualStudioVersion)</UnitTestPlatformVersion>
+
+    <!-- Set this property to override the version of the MSIX package to use -->
+    <MicrosoftWindowsSDKBuildToolsMSIXPackageVersion>1.0.20241009.2</MicrosoftWindowsSDKBuildToolsMSIXPackageVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+	  <ProjectCapability Include="TestContainer" />
+	  <PackageReference Include="Microsoft.TestPlatform" Version="17.13.0-dev" />
+	  <PackageReference Include="Microsoft.TestPlatform.ObjectModel" Version="17.13.0-dev" />
+	  <PackageReference Include="Microsoft.TestPlatform.TestHost" Version="17.13.0-dev" />
+    <PackageReference Include="Microsoft.Windows.SDK.CPP" Version="10.0.26100.2454" />
+    <PackageReference Include="MSTest.Engine" Version="1.0.0-alpha.24562.1" />
+    <PackageReference Include="MSTest.SourceGeneration" Version="1.0.0-alpha.24562.1" />
+    <PackageReference Include="MSTest.TestAdapter" Version="3.8.0-dev" />
```

Comment:
@Evangelink Is it expected to reference the three packages?

Reply:
Yea we can remove a bunch of these. See the other reference comment. Will close tis one and we can focus the discussion on the other one.

Reply:
No. We should either have source gen or regular adapter. I'll do some cleanup once we have something working.